strapi/strapi
# [Upgrade tool] Run package installer

### What does it do?

Runs the detected preferred package manager for the project after the package.json update stage

### Why is it needed?

To automatically install packages for users

TODO: the package installer runs silently and is very slow; we need to add a spinner before launch or just have it output from the PM

### How to test it?

Run the upgrade tool and see that packages are installed after being updated

Test will be added before the final feature is merged

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
